src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -2316,114 +2341,131 @@ s32 func_80833CDC(PlayState* play, s32 index) {
     }
 }
 
-void func_80833DF8(Player* this, PlayState* play) {
+/**
+ * Handles the high level item usage and changing process.
+ *
+ * Tasks include:
+ *    - Put away a mask if it is not present on any C button
+ *    - Put away an item if it is not present on the B button or any C button
+ *    - Use an item on the B button or any C button if the corresponding button is pressed
+ *    - Keep track of the curent item button being held down
+ *
+ * Note: The item put-away/take-out process is not handled here.
+ */
+void Player_ItemHandler(Player* this, PlayState* play) {
```

Comment:
I suppose. I think I was going for something less direct and more administrative, if that makes sense. since this doesnt really do any work for the item changing process itself. but it facilitates the whole button interaction thing and calls `UseItem` when it makes sense to (and puts items away if they leave c buttons). 

Maybe theres a better name for this in general, but I cant think of it

Reply:
My original ideas for the name were oriented around "updating ItemButtons", but I moved away from it because this function is directly responsible for actually using the item as well. I think `UpdateItemButtons` caters more to the "remove item if its not on a button" aspect

Reply:
my only other idea is `ProcessItemButtons` - its vague enough to cover both the case of using an item and putting it away if its not present on a button anymore, and even the `sHeldItemButtonIsHeldDown` variable

I think this is prob best tbh

Reply:
Okay yeah i changed it and really like how this looks